### PR TITLE
New header call to actions

### PR DESCRIPTION
**Pull request by zeftilldeath**
Added a secondary level of call to actions. The designs will be further experimented with but this is a good beginning. I haven't added urls as I didn't know what to put in also the data link names may not be right, please take a look and see what you think. 

Thanks

screen shot 2017-06-02 at 17 21 26

screen shot 2017-06-02 at 17 21 46

screen shot 2017-06-02 at 17 22 18

### CODE REVIEW

**Comment by paperboyo** [comment]
2173c3d2-47b8-11e7-9591-e10ca8061100

File: static/src/stylesheets/layout/new-header/_header-cta-item.scss [hunk 1]
```diff
@@ -1,64 +1,94 @@
...
+        color: currentColor;
 
-    @include mq(tablet) {
-        font-size: 18px;
-        font-weight: 900;
-        padding-left: 16px;
-        padding-right: 27px;
-        padding-top: 18px;
+        &:before {
+            border-top-color: #00d4ff;
```
Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_header-cta-item.scss]: Mimimi. This should probably go away.

File: static/src/stylesheets/layout/new-header/_header-cta-item.scss [hunk 2]
```diff
@@ -1,64 +1,94 @@
...
+            transform: rotate(-3deg) scale(1.05);
+        }
     }
 }
 
-.header-cta-item__new-line {
-    display: block;
+.header-cta-item--secondary {
+    &:not(:last-child) {
```
Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_header-cta-item.scss]: Nesting `:not` seems unnecessary.

File: static/src/stylesheets/layout/new-header/_header-cta-item.scss [hunk 3]
```diff
@@ -1,64 +1,94 @@
...
+
+.header-cta-item__label {
+    display: inline-block;
+    font-family: 'Guardian Egyptian Web', 'Guardian Text Egyptian Web', Georgia, serif;
+    font-size: 13px;
+    font-weight: 700;
+    line-height: 1;
+    padding: $gs-baseline $gs-gutter / 2;
+    position: relative;
+    z-index: 2;
```
Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_header-cta-item.scss]: Could you find a way to remove that beauty?

Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_header-cta-item.scss]: Almost certainly!

**Comment by NataliaLKB** [comment]
@gustavpursche let me know if there is anything else you think should change 😄

**Review by gu-stav — APPROVED** [review]
Looks good to me now! ✨

**Pull request merged**